linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/configure_opensc_card_drivers/rule.yml

```diff
@@ -47,8 +47,7 @@ ocil_clause: 'the smart card driver is not configured correctly'
 
 ocil: |-
     To verify that <tt>{{{ xccdf_value("var_smartcard_drivers") }}}</tt> is configured
-    as the smart card driver, run the following command changing <i>ARCH</i> for
-    the architecture of your operating system:
-    <pre>$ grep card_drivers /etc/opensc-<i>ARCH</i></pre>
+    as the smart card driver, run the following command:
+    <pre>$ grep card_drivers /etc/opensc.conf</pre>
     The output should return something similar to:
     <pre>card_drivers = {{{ xccdf_value("var_smartcard_drivers") }}};</pre>
```

Comment:
The verification steps might also use the `opensc-tool --get-conf-entry` to make it less complicated. But that is not mandatory